platform/landing-page/src/main/resources/assets/js/index.js

```diff
@@ -0,0 +1,97 @@
+var getSourceAvailabilities = function () {
```

Comment:
You'll probably want to wrap these functions in a closure.  Right now they're global in scope.